opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -555,6 +561,16 @@ def __init__(
             OTEL_SPAN_LINK_COUNT_LIMIT,
             _DEFAULT_OTEL_SPAN_LINK_COUNT_LIMIT,
         )
+        self.max_event_attributes = self._from_env_if_absent(
+            max_event_attributes,
+            OTEL_SPAN_LINK_COUNT_LIMIT,
```

Comment:
Are these placeholders until [this](https://github.com/open-telemetry/opentelemetry-specification/pull/1751) is merged? Would be good to have a TODO.

Reply:
Captured the work in an issue https://github.com/open-telemetry/opentelemetry-python/issues/1918